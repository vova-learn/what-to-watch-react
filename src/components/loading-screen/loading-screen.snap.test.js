import React from 'react';
import {render} from '@testing-library/react';
import LoadingScreen from './loading-screen';

it(`Should 'LoadingScreen' render correctly`, () => {
  const {container} = render(<LoadingScreen />);
  expect(container).toMatchSnapshot();
});
