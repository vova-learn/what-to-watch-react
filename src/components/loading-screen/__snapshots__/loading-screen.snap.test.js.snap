// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should 'LoadingScreen' render correctly 1`] = `
<div>
  <div
    class="page-content"
    style="height: 100vh; display: flex; flex-direction: column; justify-content: space-between; align-items: center;"
  >
    <div
      class="page-loading"
      style="flex-grow: 1; display: flex; align-items: center;"
    >
      <div
        class="page-loading__spinner"
      >
        <div
          class="lds-spinner"
        >
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
        </div>
      </div>
    </div>
  </div>
</div>
`;
